tests/test_file_detection.py

```diff
@@ -6,7 +6,7 @@
 
 class CSVFileDetect(TestCase):
     def test_json_file(self):
-        with open("tests/test_import_files/test_file.json", "rb") as file:
+        with open("tests/artifacts/test_file.json", "rb") as file:
```

Comment:
I don't mind renaming the folder but I feel 'artifacts' might be misinterpreted as being the outputs of a test instead of being the input files we use for testing. In other places we usually label the folder test files or fixtures so the original name isn't too far off.

Reply:
Updated to `fixtures` in https://github.com/sdelements/django-multi-import/pull/40/commits/05ac2ba37bbc4abbe189bb102c6ff3b357223772